services/121-service/src/main.ts

```diff
@@ -71,6 +73,39 @@ function generateModuleDependencyGraph(app: INestApplication): void {
   });
 }
 
+function generateSwaggerSummaryJson(app: INestApplication<any>): void {
+  const options = new DocumentBuilder()
+    .setTitle(APP_TITLE)
+    .setVersion(APP_VERSION)
+    .build();
+  const openApiDocument = SwaggerModule.createDocument(app, options);
+
+  const minimalDocument: {
+    method: string;
+    path: string;
+    returnType: string;
```

Comment:
```suggestion
    returnType?: string;
```

Instead of setting it to a hardcoded string, I would simply not set it.

Reply:
Good one